### DISCUSSION TITLE
Vercel not Recognizing My ENV Variables

### DISCUSSION THREAD

**Discussion by fahad-digisigns** [vercel/community#3048] [Help] [answered]
### Problem Description

My app is on nextjs it is working fine on local but when i hosted it on vercel i am getting the STRIPE API key as null because it is not reading my env i have added env variables in settings of Vercel i have tried re deploying multiple times but still getting the error

**Comment by amyegan** [comment]
Hi @fahad-digisigns. Did you add the [environment variables](https://vercel.com/docs/concepts/projects/environment-variables) in the project settings UI or somewhere else?

If you share a unique deployment URL or deployment id here, I can look into the logs for more information.

**Comment by bfg-coding** [comment]
@amyegan I'm having the same issue with a Nuxt project I tried to deploy yesterday.   51p885a2o  I have the environment variables setup on Vercel but it doesn't seem to be picking them up during build according to the errors I'm seeing

**Reply by amyegan** [reply, marked as answer]
For any env vars you want to use in client side code, be sure you include the `NUXT_ENV_` prefix. 

If that doesn't solve it and you need more help, please share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use for debugging. That will let me get a better look at what is going wrong with the environment variables.

**Reply by bfg-coding** [reply]
The issue resolved itself some how late Saturday night.  It was server side code and specifically in the `nuxt.config.ts` file so hard to figure out exactly how to debug the issue.     Thank you for the response though
